OtrosLogViewer-app/src/test/java/pl/otros/logview/logloader/basic/LoadingRunnableTest.java

```diff
@@ -274,14 +271,13 @@ public void testLoadStatisticAfterFullReadWithLogParser() throws Exception {
     logImporter = getJulLogParser();
     underTest = new LoadingRunnable(vfsSource, logImporter, collector, SLEEP_TIME, Optional.empty(), Optional.empty());
     saveLines(Range.closed(0, julSimpleLogLines.size()), julSimpleLogLines, outputStream);
-
+    
     final Thread thread = new Thread(underTest);
     thread.setDaemon(true);
     thread.start();
 
-    await().atMost(5, SECONDS).until(() -> underTest.getLoadStatistic().getPosition() == julSimpleAvailableBytes);
-    await().atMost(5, SECONDS).until(() -> underTest.getLoadStatistic().getTotal() == julSimpleAvailableBytes);
-    underTest.stop();
```

Comment:
Without stopping we will have thread leaks in tests.

Reply:
Stopping is also done in tearDown(). So this is duplicate code, right?

Reply:
Yes, you are right